examples/messengers/telegram/2_buttons.py

```diff
@@ -0,0 +1,170 @@
+# %% [markdown]
+"""
+# 2. Buttons
+
+
+This example shows how to display and hide a basic keyboard in Telegram.
+"""
+
+# %%
+import os
+
+import dff.script.conditions as cnd
+from dff.script import TRANSITIONS, RESPONSE
+from dff.script.core.message import Button
+from dff.pipeline import Pipeline
+from dff.messengers.telegram import (
+    PollingTelegramInterface,
+    TelegramMessenger,
+    TelegramUI,
+    TelegramMessage,
+    RemoveKeyboard,
+)
+from dff.utils.testing.common import is_interactive_mode
+
+
+# %%
+# Like Telebot, TelegramMessenger only requires a token to run.
+# However, all parameters from the Telebot class can be passed as keyword arguments.
+messenger = TelegramMessenger(token=os.getenv("TG_BOT_TOKEN", "SOMETOKEN"))
+
+
+# %% [markdown]
+"""
+To display or hide a keyboard, you can utilize the `TelegramUI` class.
+It can be initialized either with the `keyboard` parameter (allows `telebot` keyboards)
+or with the `buttons` parameter (the keyboard will be constructed under the hood).
+
+`TelegramUI` should be passed to the `ui` field
+of the generic `Response` class.
+"""
+
+
+# %%
+script = {
+    "root": {
+        "start": {
+            RESPONSE: TelegramMessage(),
+            TRANSITIONS: {
+                ("general", "native_keyboard"): (
+                    lambda ctx, actor: ctx.last_request.text in ("/start", "/restart")
+                ),
+            },
```

Comment:
RESPONSE can be removed

Reply:
https://github.com/deeppavlov/dialog_flow_framework/pull/13/commits/96710b044e50769fa7796f330399539c28e305d6